Library/Formula/hbase.rb

```diff
@@ -1,8 +1,8 @@
 class Hbase < Formula
   desc "Hadoop database: a distributed, scalable, big data store"
   homepage "https://hbase.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=hbase/1.0.1.1/hbase-1.0.1.1-bin.tar.gz"
-  sha256 "fd20fd98e9c11d96d0281077e3040c81f45bafcc1e4f14318cede31e45819fdf"
+  url "https://www.apache.org/dist/hbase/stable/hbase-1.1.2-bin.tar.gz"
```

Comment:
Can we keep using the `dyn` mirror here or is that no longer functional for `hbase`?